### PR TITLE
Added acknowledgements of RSDG, CEDA, JASMIN

### PR DESCRIPTION
**Pull request by clairbarnes**

### CODE REVIEW

File: src/components/ExtraPages/About/Acknowledgements.js [hunk 1]
```diff
@@ -8,12 +8,23 @@ export default function Acknowledgements() {
       <div>
         <h1>Acknowledgements</h1>
         <h6>
-          The development of this plot explorer has been made possible thanks to
-          the following:
+          This plot explorer was developed by UCL's <a href='https://www.ucl.ac.uk/research-it-services/research-software-development'>
```
Comment (raquelalegre) [inline src/components/ExtraPages/About/Acknowledgements.js]:
```suggestion
          This plot explorer was developed by UCL's <a href='https://www.ucl.ac.uk/advanced-research-computing/expertise/research-software-development'>
```

**Review by raquelalegre — APPROVED** [review]

**Review by raquelalegre — CHANGES_REQUESTED** [review]
These changes look good, but the tests are broken due to a dependency problem. Waiting for #68 to be merged and make sure the tests pass before approving again.

**Review by raquelalegre — APPROVED** [review]

**Comment by raquelalegre** [comment]
Tests pass now 🥳 

However the font looks a bit too bold, and there's a space between the paragraphs that looks odd. I'll try and how to make it look more like the other sections.

**Review by JamieJQuinn — APPROVED** [review]
Beautifully acknowledged!

**Pull request merged**